pkg/ebpftracer/c/tracee.bpf.c

```diff
@@ -214,6 +218,22 @@ int tracepoint__raw_syscalls__sys_exit(struct bpf_raw_tracepoint_args *ctx)
 
         id = *id_64;
     }
+
+    // Skip if cgroup is muted.
+    int zero = 0;
+    config_entry_t *config = bpf_map_lookup_elem(&config_map, &zero);
```

Comment:
For the future: We should move the cgroup option out of the map and into a constant, as this will anyway never change at runtime.